userProfile.scss

```diff
@@ -0,0 +1,44 @@
+@import './index.css';
+
+$text-color-black: #000000;
+$background-color-text: #D1D8E2;
+
+.userProfileContainer {
+  padding: 50px 10px 0 10px;
+  a {
+    text-decoration: none;
+    font-weight: bold;
+    font-size: 1.3rem;
+    color: $text-color-black;
+  }
+  h1 {
+    color: $text-color-black;
+  }
+  .backToAllUsers {
+    display: flex;
+    justify-content: flex-start;
+    padding: 40px 10px;
+  }
```

Comment:
this styling seems to be overkill just to put that in the top left corner.

 upon investigating i think i found out why you're having to do so much:
- your navbar is `position: fixed`
- that takes it out of the normal doc flow, so things end up behind it
- then you have to give those things padding to push them down so they are visible

the fix would be to have the nav with `position: sticky` instead of `fixed`:
```css
position: sticky;
top: 0;
```
this would look the same as `fixed` except it will keep it in the normal doc flow to start with, so things are not able to be behind it. so no more need to add top padding to make things visible